### DISCUSSION TITLE
Can't see any appeal button for GitHub developer pack

### DISCUSSION THREAD

**Discussion by akashperera** [community/community#136657] [GitHub Education]
### Hi everyone,

I applied for students developer pack with my university id , but it was rejected due to incomplete GitHub account , though I fix them now I can't apply with the same id , and I doesn't have any re appeal button also please help me to fix this !!!

**Comment by dewnaka7264** [comment]
Response:

Hi Akash,

It sounds like you’ve already addressed the issues that led to your initial rejection, but you’re unable to reapply or see an appeal button. Here’s what you can do to resolve this issue:

Check Your GitHub Account:

Ensure that your GitHub account is complete and meets all the necessary criteria for the GitHub Student Developer Pack. Verify that your profile is fully updated and that all required fields are filled in.
Reapply for the Pack:

If you cannot find an appeal button, it might be best to submit a new application with the updated information. Sometimes the system allows for a new application submission if the previous one was rejected.
Contact GitHub Support:

If reapplying does not work or if the appeal button is still not visible, reach out directly to GitHub Support for assistance. You can contact them through the [GitHub Support page](https://support.github.com/contact) and explain your situation. Provide them with your application ID (#136657) and details about the issue.
Visit GitHub Education Support:

GitHub Education has a specific support system for issues related to the Developer Pack. You can find additional help and resources on the GitHub Education Community or [GitHub Education Help](https://education.github.com/support).
By following these steps, you should be able to address the issue and either reapply or get the necessary support to resolve your situation.

**Comment by akashperera** [comment]
Thank you!

**Comment by davevad93** [comment]
Hi @akashperera ,

If the only reason for your initial rejection was due to incomplete profile information, by updating your profile you should be fine because your ID has already been registered as proof of your application, usually when the reason for rejection is due to academic documentation you are required to upload additional documentation to verify your student status.  Check [here](https://education.github.com/discount_requests/application) your status, under “***You have already submitted X applications***” what does it say?

**Reply by akashperera** [reply]
In my account I have connected two email addresses from two universities , A year ago I activated the developer pack with one email ID, but it was expired last month. my new id with new expire date was misplaced so I tried with my other univerisity ID. so when I apply it was rejected. There was billing info issue so I filled it and relogged and tried, it gives an error saying that the ID is already submitted and cannot apply with the same binary info


It says "You've already submitted 3 requests"

**Reply by davevad93** [reply]
I see @akashperera , that's weird because if you received your student benefits last year they shouldn't have expired because they last for 2 years. 

Anyway there's one thing I didn't quite understand, the error referring to your "ID" does it refer to your university ID card or to your university email address?

If it's the latter, the problem is that the `Student Developer Pack` benefits are linked to the educational email address and both of your emails count as "already used" for GitHub Education even though your second email hasn't actually been used because your request has been rejected; something similar happened here #126637 , I think you should contact support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

**Comment by rajmssuperhost** [comment]
hh

**Comment by rajmssuperhost** [comment]
g
